### DISCUSSION TITLE
Can someone provide a brief snippet on customizing controllers in v6?

### DISCUSSION THREAD

**Discussion by llaalgol** [fabricjs/fabric.js#10026] [Q&A] [answered]
I just migrated to v6, and my app works fine after updating deprecated code, except for the controllers. I'm unsure how to customize them. I would appreciate your help

**Comment by asturur** [comment, marked as answer]
If you mean Controls, please have a look here:

https://fabricjs.github.io/demos/custom-controls/

and here

https://fabricjs.github.io/docs/configuring-controls/
